cli/mender-artifact/read.go

```diff
@@ -179,11 +181,15 @@ func printPayload(index int, p handlers.Installer) {
 		}
 	}
 
-	for _, f := range p.GetUpdateAllFiles() {
+	if len(p.GetUpdateAllFiles()) == 0 {
+		fmt.Printf("    Files: None\n")
+	} else {
 		fmt.Printf("    Files:\n")
```

Comment:
Question: The `Files:` header was removed at one point, but not here. Why?

Reply:
It was not removed, it was just prevented from being printed over and over (previously it was inside the loop).